src/main/car/inventory/DealerGroup.java

```diff
@@ -2,19 +2,28 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
 public class DealerGroup {
 
 	private ArrayList<Dealership> dealers;
-	
+
 	public DealerGroup() {
 		this.dealers = new ArrayList<>();
 	}
 
-	public ArrayList<Dealership> getDealers() {
+	public ArrayList<Dealership> getDealergroup() {
 		return dealers;
 	}
-	
+
```

Comment:
this method is not necessary, either remove it or rename it. getDealers should not return a String